### ISSUE TITLE
Tensorflow implementation with  the 2stream evaluation code

### ISSUE THREAD

**Issue by jpainam** [layumi/Person-reID_GAN#14]
Hi!
I want to know if you do have a tensorflow implementation of this paper. Or a repository to recommend. I'm most interested with the 2stream evaluation code.
How to implement the LRSO loss function when using [verification+identification](https://github.com/layumi/2016_person_re-ID) model
Thank you.

**Comment by XN-0808** [comment]
> Hi!
> I want to know if you do have a tensorflow implementation of this paper. Or a repository to recommend. I'm most interested with the 2stream evaluation code.
> How to implement the LRSO loss function when using [verification+identification](https://github.com/layumi/2016_person_re-ID) model
> Thank you.

请问你现在有完成TensorFlow版的实现吗？特别是关于LSRO部分的反向传播问题
